formspree/forms/models.py

```diff
@@ -230,7 +235,7 @@ def send(self, data, keys, referrer):
                     data=data, host=self.host, keys=keys, now=now,
                     unconfirm_url=unconfirm)
         else:
-            if monthly_counter - settings.MONTHLY_SUBMISSIONS_LIMIT > 25:
+            if monthly_counter - monthly_limit > 25:
```

Comment:
Uh, doesn't this mean that we'll send the overlimit notification only after they've gone 25 submissions past the limit? And continuously from then on? Shouldn't it be (overlimit and monthly_counter - monthly_limit < 25) ??

Reply:
That's true. Another reason to merge this ASAP.

Reply:
I mean, after the fix.

Reply:
👍

Reply:
We should probably set the new env vars before merging

Reply:
This has been wrong for so long that all tests were wrong and now are failing.

Reply:
No, it wasn't wrong.

Reply:
That `if` block was for returning from the function without sending any emails, that would happen after the 25 past the limit. For the first 25 past the limit we the function would continue and an email would be sent with the overlimit notification contents instead of the form data.

But no one can disagree it was very confusing.